src/Akeneo/Pim/Enrichment/Product/back/Test/Specification/API/Command/UserIntent/SetAssetValueSpec.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Pim\Enrichment\Product\Test\Specification\API\Command\UserIntent;
+
+use Akeneo\Pim\Enrichment\Product\API\Command\UserIntent\SetAssetValue;
+use Akeneo\Pim\Enrichment\Product\API\Command\UserIntent\ValueUserIntent;
+use PhpSpec\ObjectBehavior;
+
+/**
+ * @copyright 2022 Akeneo SAS (https://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class SetAssetValueSpec extends ObjectBehavior
+{
+    function let()
+    {
+        $this->beConstructedWith('code', 'ecommerce', 'en_US', ['absorb_packshot_1', 'admete_packshot_3']);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType(SetAssetValue::class);
+        $this->shouldImplement(ValueUserIntent::class);
+    }
+
+
+    function it_returns_the_attribute_code()
+    {
+        $this->attributeCode()->shouldReturn('code');
+    }
+
+    function it_returns_the_locale_code()
+    {
+        $this->localeCode()->shouldReturn('en_US');
+    }
+
+    function it_returns_the_channel_code()
+    {
+        $this->channelCode()->shouldReturn('ecommerce');
+    }
+
+    function it_returns_the_value()
```

Comment:
```suggestion
    function it_returns_the_values()
```

c'est bien pour trouver un truc à redire :smile: :trollface: